pages/sequential.jsx

```diff
@@ -78,9 +79,14 @@ module.exports = React.createClass({
     if (!this.state.showCvcHint) {
       cvcHintClassName += " hidden";
     }
+    // FIXME: Right now we only display this header for `en` locales
+    // this should be change to display for all once we have other locales translated.
+    if (/^(en)(\b|$)/.test(this.props.locales[0])) {
```

Comment:
If we decide to keep this, possible we can build it into the Header component so we don't need to have the same code in two spots? You might have to pass in locale[0] to do the check, but I'm fine with that.